lib/hammer_cli_foreman/attribute.rb

```diff
@@ -0,0 +1,303 @@
+require 'hammer_cli_foreman/image'
+require '../hammer-cli-foreman/lib/hammer_cli_foreman/compute_resource/register_compute_resources'
+require '../hammer-cli-foreman/lib/hammer_cli_foreman/compute_resource/help_utils'
+
+module HammerCLIForeman
+  class Attribute < HammerCLIForeman::Command
+    resource :compute_attributes
+
+    def self.get_params(options)
+      params = {}
+      params['compute_attribute'] = {}
+      profile = HammerCLIForeman.record_to_common_format(
+          HammerCLIForeman.foreman_resource(:compute_profiles).call(:show, 'id' => options['option_compute_profile_id'] )
+      )
+      params['compute_attribute'] = profile['compute_attributes'].select { |hash| hash['compute_resource_id'] == options['option_compute_resource_id']}[0] || {}
+      params['compute_attribute'].delete('attributes') if params['compute_attribute']['attributes']
+      params
+    end
+
+    def self.attribute_hash(attribute_list)
+      attribute_list.size.times.map { |idx| idx.to_s }.zip(attribute_list).to_h
+    end
+
+    class Create < HammerCLIForeman::CreateCommand
+        desc _('Create compute profile set of values.')
```

Comment:
The command descriptions shouldn't end with '.'. Please see the test results for other occurrences.

Reply:
done